manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -49,6 +49,60 @@ spec:
         prometheus.io/path: "/stats/prometheus"
         {{- end }}
         sidecar.istio.io/inject: "false"
+        {{- if not (eq .Release.Namespace "istio-system") }}
+        proxy.istio.io/config: |-
+          {{- if .Values.global.meshID }}
```

Comment:
I don't think we want to copy the entire thing here, as we will miss new fields. Also, I don't think this is correct as it doesn't handle the merging the we do in the real mesh config with `values.meshConfig`.

I think the way around this is to use templates in helm which I guess doesn't work with multiple charts... but I guess we could at least have a makefile component that syncs them (we do it for istio-remote today)